sources/packages/backend/libs/integrations/src/services/ssh/sftp-integration-base.ts

```diff
@@ -66,6 +66,112 @@ export abstract class SFTPIntegrationBase<DownloadType> {
     }
   }
 
+  /**
+   * Converts input into an ASCII 7 bit buffer.
+   * @param rawContent Raw content in string format.
+   * @returns a buffer of the input with extended ASCII accented characters converted to unaccented characters.
+   */
+  convertRawContent(rawContent: string) {
```

Comment:
If the method is not needed outside it can be declared as `private`.

Reply:
updated